buildscripts/semantic-convention/templates/SemanticAttributes.scala.j2

```diff
@@ -69,7 +69,7 @@ object {{class}} {
   /**
    * The URL of the OpenTelemetry schema for these keys and values.
    */
-  val SchemaUrl: String = "{{schemaUrl}}"
+  final val SchemaUrl: String = "{{schemaUrl}}"
```

Comment:
For a constant value to be inlined, you must remove the type annotation in addition to adding `final`.

```suggestion
final val SchemaUrl = "{{schemaUrl}}"
```

---

> A constant value definition is of the form
> ```scala
> final val x = e
> ```
> where `e` is a [constant expression](https://www.scala-lang.org/files/archive/spec/2.13/06-expressions.html#constant-expressions). The final modifier must be present and no type annotation may be given. References to the constant value `x` are themselves treated as constant expressions; in the generated code they are replaced by the definition's right-hand side `e`.

https://www.scala-lang.org/files/archive/spec/2.13/04-basic-declarations-and-definitions.html#value-declarations-and-definitions